### PR TITLE
Deprecate the repository and redirect to OCP documentation

### PR DESCRIPTION
**Pull request by ghost**
@Preeticp

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+= Openshift Pipelines Documentation
+
+WARNING: This documentation repo has been deprecated. Please visit the official OpenShift Container Platform (OCP) documentation at link:https://docs.openshift.com/container-platform/4.4/welcome/index.html[OCP 4.4 Documentation]
```
Comment (Preeticp) [inline README.adoc]: Let's link to the exact pipeline docs

Comment (Preeticp) [reply README.adoc]: https://docs.openshift.com/container-platform/latest/pipelines/understanding-openshift-pipelines.html

Comment (Preeticp) [reply README.adoc]: OCP 4.4 Documentation change this to OpenShift Pipelines documentation

Comment (ghost) [reply README.adoc]: (latest + an exact pipelines docs (understanding pipelins)

**Pull request merged**